core/src/main/java/com/yahoo/ycsb/GoodBadUglyDB.java

```diff
@@ -1,12 +1,12 @@
-/**                                                                                                                                                                                
+/**
```

Comment:
Sorry about. I already changed that in my new commit.
My IDE had import optimization enabled.